File: Cup Finals.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int x=sc.nextInt();
            int y=sc.nextInt();
            int d=sc.nextInt();
            if(Math.abs(x-y)<=d)
            System.out.println("YES");
            else
            System.out.println("NO");
        }
    }
}

File: Cup Finals.c
#include<stdio.h>
#include<math.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int x,y,d;
        scanf("%d%d%d",&x,&y,&d);
        if(abs(x-y)<=d)
        printf("YES\n");
        else
        printf("NO\n");
    }
}
